src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonapi/PrisonApiClient.kt

```diff
@@ -59,13 +59,15 @@ class PrisonApiClient(private val prisonApiWebClient: WebClient) {
   fun getPrisonersAppointmentsAtLocations(prisonCode: String, prisonerNumber: String, onDate: LocalDate, vararg locationIds: Long): List<PrisonerSchedule> = if (locationIds.isNotEmpty()) {
     log.info("PRISON-API CLIENT: query params - prisonCode=$prisonCode, prisonerNumber=$prisonerNumber, onDate=$onDate, locationIds=${locationIds.toList()}")
     getPrisonersAppointments(prisonCode, prisonerNumber, onDate)
-      .also { log.info("PRISON-API CLIENT: matches pre-location filter: $it") }
+      .also { log.info("PRISON-API CLIENT: matches pre-location filter: ${it.map { ps -> ps.obfuscated() }}") }
       .filter { it.locationId in locationIds }
-      .also { log.info("PRISON-API CLIENT matches post-location filter: $it") }
+      .also { log.info("PRISON-API CLIENT matches post-location filter: ${it.map { ps -> ps.obfuscated() }}") }
   } else {
     emptyList()
   }
 
+  private fun PrisonerSchedule.obfuscated() = copy(firstName = "XXX", lastName = "XXX")
```

Comment:
Ensure names are not logged.

Unfortunately this is one of the downsides of using data class, by default data classes provide a `toString() `implementation which will include all fields/attributes.